ntbs-service/deployments/test.yml

```diff
@@ -42,7 +42,7 @@ spec:
                 name: migration-connection-string
                 key: connectionString
           - name: AdfsOptions__Wtrealm
-            value: "https://ntbs-test.da0c702c3b574e3185f6.uksouth.aksapp.io"
+            value: "https://ntbs-test.da0c702c3b574e3185f6.uksouth.aksapp.io/"
```

Comment:
do we need to fix this in the `uat.yml` file?

Reply:
I will merge this, make changes in ADFS vm and if everything works on both int and uat, I will do same for uat.

Reply:
curious what the changes in ADFS VMs are that are needed. Let's make sure we update the documentation we have for that and maybe link to the documentation changes in this PR too?